projects/docs/src/app/layout/components/menu/menu.component.html

```diff
@@ -0,0 +1,18 @@
+<nav class="max-h-full p-2">
+  <ul class="space-y-1">
+    @for (item of componentsMenu; track $index) {
+      <li>
+        <a
+          ckButton
+          size="sm"
+          appearance="ghost"
+          class="w-full justify-start"
+          routerLinkActive="bg-muted"
```

Comment:
try not to force a class on the component, but alter its' existing properties. In this case I'd change the value of `appearance` depending on whether the link is active or not